### DISCUSSION TITLE
[Feature Request] More features for profiles

### DISCUSSION THREAD

**Discussion by Le00nn** [LBPUnion/ProjectLighthouse#447] [Ideas]
**Adding the following to profiles on website would be quite neat:**
• Seeing the total play count for all your published levels
• Seeing the levels on profile
• Ability to change your biography on website
• Ability to change manage your published levels - edit details and deleting levels
• See join date
• Ability to see who you have hearted
• Tab for profile and privacy settings

**Comment by Ezoiar** [comment]
I would like to add additional ideas here.

- Extra badges in-website (e.g. Beta Tester)
- Offline since
- Interactable links in descriptions
- Sortable list of levels, This goes after the feature of seeing levels in profiles is added.
- Online Playtime?
- Why not add a settings page?
